variables.tf

```diff
@@ -27,6 +27,12 @@ variable "branch" {
   description = "The git branch to trigger the TFE workspace for"
 }
 
+variable "connect_vcs_repo" {
+  type        = bool
+  default     = true
+  description = "Whether to connect a VCS repo to the workspace"
+}
```

Comment:
Please reuse the existing variables `repository_owner` and `repository_name` to detect if it's needed to connect.

And additionally, can you share the background of why you need this? What is the point of having a workspace that is not connected to a VCS?

Reply:
I would like to trigger the workspace from a Github Workflow because I want to be able to feedback on the workflow results there. Gives me more control over when and how I trigger it. I will make the changes suggested.